plenum/test/waits.py

```diff
@@ -107,7 +107,7 @@ def expectedClientRequestPropagationTime(nodeCount):
 
 
 def expectedTransactionExecutionTime(nodeCount):
-    return int(CLIENT_REPLY_TIMEOUT * nodeCount)
+    return int(2 * CLIENT_REPLY_TIMEOUT * nodeCount)
```

Comment:
It fixes https://evernym.atlassian.net/browse/SOV-947
Anyway timeouts were set just from the top of the head. So they are meant to be adjusted.